src/index.html

```diff
@@ -11,8 +14,86 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/index.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <!--<h1>Product cards</h1>-->
+
+    <div
+      class="bloc"
+      data-qa="card"
+    >
+      <img
+        class="bloc__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      />
+
+      <div class="bloc__text1">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="bloc__text2">Product code: 195434</div>
+
+      <div class="bloc__review">
+        <div class="bloc__review-stars">
+          <img
+            class="bloc__review-stst"
+            src="images/star-active.svg"
+            alt="starA"
+          />
+          <img
+            class="bloc__review-stst"
+            src="images/star-active.svg"
+            alt="starA"
+          />
+          <img
+            class="bloc__review-stst"
+            src="images/star-active.svg"
+            alt="starA"
+          />
+          <img
+            class="bloc__review-stst"
+            src="images/star-active.svg"
+            alt="starA"
+          />
+          <img
+            class="bloc__review-stst"
+            src="images/star.svg"
+            alt="starUn"
+          />
+        </div>
```

Comment:
You need to use this block from the previous task and add the modifier class
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```